gems/aws-sdk-s3/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 Unreleased Changes
 ------------------
 
+* Feature - Expose presigned request status to the request handler stack #2513
```

Comment:
I think it makes sense to classify this as an issue instead - the intended feature has a gap.